<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyDialog</class>
 <widget class="QWidget" name="ApplyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>469</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>469</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>申请加课</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    border-width: 1px;
    padding: 1px;
    border-style: solid;
	border-color: rgb(22, 102, 57);
    border-radius: 5px;
}

/* As mentioned above, eliminate the padding and increase the border. */
QLineEdit:focus:focus {
    border-width: 3px;
    padding: 0px;
}
#ApplyDialog{
background-color: rgb(235, 244, 234);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>241</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>考查方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kcfscomboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="addresslabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上课地点</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="capacitylabel">
      <property name="text">
       <string>开设人数</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="capacitylineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>课程课时</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="timelineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>41</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="deplabel">
      <property name="text">
       <string>开设学院</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="namelabel">
      <property name="text">
       <string>课程名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namelineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>课程id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="typelabel">
      <property name="text">
       <string>课程性质</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="classifycomboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cscorelabel">
      <property name="text">
       <string>课程学分</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="scorelineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="deplineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="idlineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>280</y>
     <width>230</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox1"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox2"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox3"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
